pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1281,7 +1285,7 @@ func (t *Task) hasAllProgressReportingCompleted() (bool, error) {
 			Message:  pendingMessage,
 		})
 	}
-	return !isAnyRunning && !isAnyPending && isCompleted, nil
+	return !isAnyRunning && !isAnyPending && isCompleted && !isAnyUnknown, nil
```

Comment:
nit: switch isAnyUnknown and isCompleted so all the negatives are before the one that is not?

Might not be a real thing just a thought

Reply:
+1

Reply:
Updated